slices/dosfstools.yaml

```diff
@@ -0,0 +1,24 @@
+package: dosfstools
+
+essential:
+  - dosfstools_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+    contents:
```

Comment:
same here: can we add more test coverage pls :)

Reply:
It seems there is no need for tests for the bins other than `/usr/sbin/fatlabel`, `/usr/sbin/fsck.fat` and `/usr/sbin/mkfs.fat`, as the other files in `/usr/sbin/` are just symlinks to these three files.

Reply:
Noted